Public/Src/Engine/Scheduler/Artifacts/FileContentManager.cs

```diff
@@ -2818,7 +2818,7 @@ private string GetAssociatedPipDescription(FileOrDirectoryArtifact declaredArtif
                 if (declaredArtifact.FileArtifact.IsSourceFile)
                 {
                     consumerDescription = m_host.GetConsumerDescription(declaredArtifact);

```

Comment:
>GetConsumerDescription [](start = 49, length = 22)

This is the only place where this method is called. Maybe change its behavior to return null if there is more than one consumer (and update comment on IFileContentManagerHost.GetConsumerDescription to represent new behavior of the API.